src/WebJobs.Script/OutOfProc/ProcessManagement/WorkerDescription.cs

```diff
@@ -28,14 +28,6 @@ public abstract class WorkerDescription
         /// </summary>
         public List<string> Arguments { get; set; }
 
-        public virtual void FixAndValidate()
-        {
-            WorkerDirectory = WorkerDirectory ?? Directory.GetCurrentDirectory();
-            Arguments = Arguments ?? new List<string>();
-            if (!string.IsNullOrEmpty(DefaultWorkerPath) && !Path.IsPathRooted(DefaultWorkerPath))
-            {
-                DefaultWorkerPath = Path.Combine(WorkerDirectory, DefaultWorkerPath);
-            }
-        }
+        public abstract void FixAndValidate(string workerDirectory);
```

Comment:
>public abstract void FixAndValidate(string workerDirectory); [](start = 8, length = 60)

nit: perhaps a one liner on what this does?

Reply:
renamed